tests/python-pytest/onnx/import/mxnet_backend_rep.py

```diff
@@ -69,30 +69,15 @@ def run(self, inputs, **kwargs):
         data_names = [graph_input for graph_input in self.symbol.list_inputs()
                       if graph_input not in self.arg_params and graph_input not in self.aux_params]
 
-        data_shapes = []
-        for idx, input_name in enumerate(data_names):
-            data_shapes.append((input_name, inputs[idx].shape))
+        if self.arg_params:
```

Comment:
is this duplicate of  MXNetBackendRep in  `tests/python-pytest/onnx/export/backend_rep.py` ?

Reply:
Agree. Will create a common file in `tests/python-pytest/onnx/` and make changes in other files accordingly.